### DISCUSSION TITLE
Got a 'BT::RuntimeError' what(): Missing manifest.

### DISCUSSION THREAD

**Discussion by aarontrain** [BehaviorTree/BehaviorTree.CPP#684] [Q&A]
Hi, I've got a 'BT::RuntimeError' whose details are as follows:
"terminate called after throwing an instance of 'BT::RuntimeError'
what(): Missing manifest. It shouldn't happen. Please report this issue"

Maybe I can get your help

**Comment by facontidavide** [comment]
Which version is this? 3.8?

**Reply by aarontrain** [reply]
Yes, runtime.txt shows it is the v3.8.

**Comment by facontidavide** [comment]
I will need WAAAAAAAY more information to debug this.
Do you have any example to reproduce the issue?

**Reply by aarontrain** [reply]
The problem arises in a project that **uses behavior trees to model entities in discrete event simulations**, which is somewhat complex to describe.

Maybe you can shed some light on where the bug might be, or I'll get back to you after I upload the project to GitHub.
